shell/platform/fuchsia/dart/BUILD.gn

```diff
@@ -108,17 +123,232 @@ dart_library("litetest") {
   ]
 }
 
-dart_library("meta") {
-  package_root = "//third_party/dart/pkg/meta"
-  package_name = "meta"
+dart_library("args") {
+  package_root = "//third_party/dart/third_party/pkg/args"
+  package_name = "args"
 
   language_version = "2.12"
 
   deps = []
 
   sources = [
-    "dart2js.dart",
-    "meta.dart",
-    "meta_meta.dart",
+    "args.dart",
+    "command_runner.dart",
+    "src/allow_anything_parser.dart",
+    "src/arg_parser.dart",
+    "src/arg_parser_exception.dart",
+    "src/arg_results.dart",
+    "src/help_command.dart",
+    "src/option.dart",
+    "src/parser.dart",
+    "src/usage.dart",
+    "src/usage_exception.dart",
+    "src/utils.dart",
+  ]
+}
+
+dart_library("collection") {
```

Comment:
What is using these new Dart dependencies? I don't see references to them in the test .dart code that was added. Why does adding tests increase the dependencies of our platform in general?

Reply:
I see, I missed this part of the description:

> Note this PR also migrates dart_library dependencies from flutter/buildroot (secondary/third_party/... packages) to a central flutter fuchsia BUILD.gn file, so the secondary BUILD.gn files that originally defined those dart_library targets can now be removed.

Might be a good candidate for splitting into a separate pull request, but up to you.

Reply:
Actually, there are Fuchsia SDK APIs that depend on these. All of the dependencies are in fact in use (when including the integration test).

Reply:
See https://github.com/flutter/engine/pull/26880/files/7b5ce3b1b3504c9ad1fe5330df1ed48784c64485#diff-6378baabb0a230513a31db033afa400fcb2afea278438394af653d525d1b105aR66

Reply:
I could break this PR into 2 or three separate PRs, but if there is no objection, the changes do fit together well enough to keep them together.